photon-ml/src/integTest/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinateTest.scala

```diff
@@ -55,17 +58,17 @@ class RandomEffectCoordinateTest extends SparkTestUtils with GameTestUtils {
   @Test(dataProvider = "numEntitiesDataProvider")
   def testScore(numEntities: Int): Unit = sparkTest("testScore") {
     val (coordinate, model) = generateRandomEffectCoordinateAndModel(
-      randomEffectId, featureShardId, numEntities, NumTrainingSamples, Dimensionality)
+      randomEffectId, featureShardId, numEntities, NUM_TRAINING_SAMPLES, DIMENSIONALITY)
 
     val score = coordinate.score(model)
 
-    assertEquals(score.scores.count, numEntities * NumTrainingSamples)
+    assertEquals(score.scores.count, numEntities * NUM_TRAINING_SAMPLES)
     assertTrue(score.scores.map(_._2).collect.forall(_ == 0.0))
   }
 }
 
 object RandomEffectCoordinateTest {
-  val NumTrainingSamples = 1000
-  val Dimensionality = 10
-  val NumPartitions = 4
+  val NUM_TRAINING_SAMPLES = 1000
+  val DIMENSIONALITY = 10
+  val NUM_PARTITIONS = 4
```

Comment:
Regarding the code style issue, actually I guess it might be a good idea to follow [Spark's Scala code style for](https://github.com/databricks/scala-style-guide), instead of the Scala style guide, if we are still considering to contribute some portion of Photon ML back to Spark.

However, please note that Spark's guide on Scala code style is closer to the Java convention and somewhat different from the Scala style guide, for example, [Spark prefers the constants to be in uppercases](https://github.com/databricks/scala-style-guide#syntactic-style).

There are a few noticeable differences (and I should've mentioned them earlier), for example, the [doc style](https://github.com/databricks/scala-style-guide#doc).

Actually this is why you may have noticed that the code style of Photon ML sometimes doesn't follow the Scala code style guide strictly.